factory/network/networkComponents.go

```diff
@@ -99,13 +105,15 @@ func NewNetworkComponentsFactory(
 	if check.IfNil(args.CryptoComponents) {
 		return nil, errors.ErrNilCryptoComponentsHolder
 	}
-	if args.NodeOperationMode != common.NormalOperation && args.NodeOperationMode != common.FullArchiveMode {
+	if args.NodeOperationMode != common.NormalOperation &&
```

Comment:
perhaps we should keep only one NodeOperationMode for both, more details in the next comment